### PR TITLE
Gun Renovation +misc fixes

### LINKED ISSUE #1459: Toploader 5.56 magazines are invisible and can't be loaded into guns
Wasted like 10 TC on these fuckers before I realised they weren't showing up.

### LINKED ISSUE #1475: Orbiting someone only throws up a message to other admins when you stop orbiting
Something i ported probably

### PR DESCRIPTION
**Pull request by Chronitonity**
Its 3 AM and I used my master and I blame Whitefox for all of this

Thanks to Lancartwright from TG for the bioterror and the Gun renovation

Basically since we removed bulldogs all the guns are much cheaper now for Nuke Ops. All of these numbers are negotiable obviously but I think these work pretty well. 

You can also buy big bundles of a gun+ammo for cheaper.

The L6 Saw is cheaper again, ammo is cheaper, and requires an open hand to fire, but gets new ammo types
Bioterror Foam nades added as well, they function as 
Orbiting, Assault Pods, and Medbeam fixes too.

Fixes #1459 
Fixes #1475 
As I said, this is my master. It'd be chill if this could get a quick look at.

### CODE REVIEW

**Comment by Lawligagger** [comment]
Looks good to me.

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1378,14 +1378,19 @@ B --><-- A
 	shift.Translate(0,radius)
 	transform = shift
 	SpinAnimation(rotation_speed, -1, clockwise, rotation_segments)
-	while(orbiting && orbiting.loc)
-		var/targetloc = get_turf(orbiting)
+
+	//we stack the orbits up client side, so we can assign this back to normal server side without it breaking the orbit
+	transform = initial_transform
+	while(orbiting && orbiting == A && A.loc)
+		var/targetloc = get_turf(A)
```
Comment (danhibiki) [inline code/__HELPERS/unsorted.dm:1385]: this fixes the orbitting bug?

Comment (Chronitonity) [inline code/__HELPERS/unsorted.dm:1385]: Yeah there were a few unreported ones it should address as well

**Comment by SplergeBucket** [comment]
looks good

File: code/datums/uplink_item.dm [hunk 2]
```diff
@@ -512,31 +568,58 @@ var/list/uplink_items = list()
 	name = "Toploader Magazine - 5.56"
```
Comment (danhibiki) [inline code/datums/uplink_item.dm:568]: This supposidely also has a missing sprite, not related to this PR but of note

File: code/modules/projectiles/guns/projectile.dm [hunk 3]
```diff
@@ -165,4 +165,10 @@
 	desc = "A universal syndicate small-arms suppressor for maximum espionage."
 	icon = 'icons/obj/guns/projectile.dmi'
 	icon_state = "suppressor"
-	w_class = 2
\ No newline at end of file
+	w_class = 2
+
+ /obj/item/weapon/suppressor/specialoffer
+	name = "cheap suppressor"
+	desc = "A foreign knock-off suppressor, it feels flimsy, cheap, and brittle. Still fits all weapons."
```
Comment (danhibiki) [inline code/modules/projectiles/guns/projectile.dm]: truly probably my most nitpicky thing, but the base one doesn't fit all guns either. Probably should say some.

**Comment by Foosili** [comment]
good shit

File: code/datums/uplink_item.dm (hunk 2 above)
Comment (Chronitonity) [inline code/datums/uplink_item.dm:568]: Fixed

**Pull request merged**